File: tests/strings/parse-bool.js
export function parseBoolean$fromString$(str) {
  return str === "True" || str !== "False" && null;
}
export function parseBooleanTrue() {
  return true;
}
export function parseBooleanFalse() {
  return false;
}
export function parseBooleanElse() {
  return null;
}

File: tests/strings/parse-bool.swift
public func parseBoolean(fromString str: String) -> Bool? {
	return Bool(str)
}

public func parseBooleanTrue() -> Bool? {
	return Bool("True")
}

public func parseBooleanFalse() -> Bool? {
	return Bool("False")
}

public func parseBooleanElse() -> Bool? {
	return Bool("Else")
}
